### PR TITLE
Track critical myCollection events and contexts

### PR DESCRIPTION
**Pull request by mikehrom**
- groups add, edit, delete and tap events for the My Collection flow
- adds contexts and owners for myCollection and myCollectionArtwork
- specs [here](https://www.figma.com/file/yDSHlbJVzBxvSUROD2bfRI/Sell-tab?node-id=1%3A2) and [here](https://www.figma.com/file/yDSHlbJVzBxvSUROD2bfRI/Sell-tab?node-id=466%3A24999)

### CODE REVIEW

File: src/Schema/Events/MyCollection.ts
Comment (eessex) [inline src/Schema/Events/MyCollection.ts:28]: We usually collect `slug` as well as `id` when applicable, why not in this case?

Comment (mikehrom) [reply src/Schema/Events/MyCollection.ts:28]: good q. i don't know if collections in 'My Collection' will even have slugs, but I assume they will have an id, which i left optional.

Comment (mikehrom) [reply src/Schema/Events/MyCollection.ts:28]: could throw it in as `?` optional to be consistent

Comment (eessex) [reply src/Schema/Events/MyCollection.ts:28]: Most records with an `id` should also have a slug, but not sure of the details for an individual user's collection. I do wonder if we want to know the `owner_id` and `owner_slug` of the collected artwork?

Comment (mikehrom) [reply src/Schema/Events/MyCollection.ts:28]: i left details about the collected artwork off of this event because it's possible a user can tap 'add artwork' and then exit the flow before submitting, but i suppose in those instances, 'destination' info would be null, while for completed submissions, the destination info would be present.

Comment (mikehrom) [reply src/Schema/Events/MyCollection.ts:28]: actually, i take that back. the event will fire before we know anything about the added artwork, so i think in this case, we won't need 'destination' info.

Comment (mikehrom) [reply src/Schema/Events/MyCollection.ts:28]: (there should be a backend event recorded for a submitted artwork to My Collection)

**Comment by mikehrom** [comment]
Screen Shot 2020-11-30 at 12 56 53 PMScreen Shot 2020-11-30 at 12 57 11 PM

For these highlighted links, I'm wondering if we can use TappedShowMore with the appropriate subject, context, and destination information. Feels unideal to make different events for each, though I submit the blue-boxed button is a bit unique in that it directs users to the current sell tab flow. What do you guys think? @samchieng @brainbicycle

**Comment by samchieng** [comment]
@mikehrom i would agree with that, for the red boxes i think we just want to get a rough idea if users are exploring more info on this page—so don't need to make different events for each. But separating out the blue box would give us insight into potential sell conversion.

**Review by brainbicycle — APPROVED** [review]

**Comment by brainbicycle** [comment]
LGTM, feel free to merge when ready! Not sure if you are waiting on any other feedback

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.64.0`](https://github.com/artsy/cohesion/releases/tag/v1.64.0) :rocket:

**Pull request merged**
